### PR TITLE
Few improvments

### PR DESCRIPTION
**Pull request by 4miners**

### CODE REVIEW

**Comment by Isabello** [comment]
This all looks great!

We probably want to retain our version of this commit since it forces https for both the map and the accreditation:

 https://github.com/LiskHQ/lisk-explorer/pull/21/commits/27baa634b306225f080ab3cb96f555764f4e4db4

File: sockets/delegateMonitor.js [hunk 1]
```diff
@@ -130,54 +168,123 @@ module.exports = function (app, connectionHandler, socket) {
...
+            return cb('getNextForgers (already running)');
+        }
+        running.getNextForgers = true;
+        delegates.getNextForgers(
+            function (res) { running.getNextForgers = false; cb('NextForgers'); },
+            function (res) { running.getNextForgers = false; cb(null, res); }
+        );
+    };
+
+    var getLastBlocks = function (results, init) {
```
Comment (4miners) [inline sockets/delegateMonitor.js]: That function looks sad - there is some callback hell there. It works ofc, but should be reviewed.

File: sockets/networkMonitor.js [hunk 2]
```diff
@@ -34,9 +34,10 @@ module.exports = function (app, connectionHandler, socket) {
                 log('Emitting new data');
                 socket.emit('data', data);
 
-                newInterval(0, 10000, emitData1);
-                newInterval(1, 30000, emitData2);
-                newInterval(2, 60000, emitData3);
+                newInterval(0, 1000, emitData1);
+                // FIXME: Here we are pulling 8640 blocks - logic should be changed
+                newInterval(1, 300000, emitData2);
+                newInterval(2, 1000, emitData3);
```
Comment (4miners) [inline sockets/networkMonitor.js]: We can pull here every sec, performance impact is negligible, and that looks great when things changing in realtime.

**Comment by Isabello** [comment]
Reviewing these changes now. I have on a public node to running this now to help get them tested by a wider audience.

**Comment by Isabello** [comment]
I manually merged this PR. Not sure how to fix the blame. Sorry.

**Pull request closed without merging**